starter-core/src/test/groovy/io/micronaut/starter/feature/build/MicronautAotBuildPluginSpec.groovy

```diff
@@ -13,7 +13,7 @@ import static io.micronaut.starter.options.BuildTool.GRADLE_KOTLIN
 
 class MicronautAotBuildPluginSpec extends ApplicationContextSpec implements CommandOutputFixture {
 
-    private static final String GRADLE_PLUGIN_VERSION = '3.2.2'
+    private static final String GRADLE_PLUGIN_VERSION = '3.3.0'
```

Comment:
this should not have the plugin version harcoded, instead it should retrieve it from the pom.xml

Reply:
It is in a test. nevermind.